brd-ios/breadwallet/src/Redesign/Core/CLEANerSwift/VIP/Coordinator.swift

```diff
@@ -171,9 +171,7 @@ class BaseCoordinator: NSObject,
         coordinator.start()
         coordinator.parentCoordinator = self
         
-        if let vc = coordinator.navigationController.children.first(where: { $0 is DeleteProfileInfoViewController }) as? DeleteProfileInfoViewController {
-            vc.dataStore?.keyMaster = keyMaster
-        }
+        (coordinator.navigationController.children.first as? DeleteProfileInfoViewController)?.dataStore?.keyMaster = keyMaster
```

Comment:
what would be cleaner is to add a
start(with keyMaster: Keymasster?) 
method to deleteCoordinator.. and handle the passing in there

(sry didnt think bout that before)